### PR TITLE
Merge Official and Community into Apps

### PR DESCRIPTION
**Pull request by maniackcrudelis**
Following https://github.com/YunoHost/apps/pull/677

This PR propose to merge Official and Community into one list simply named "Apps"
Official apps are tagged as `working` instead of `validated` and get by the way the flag `high_quality`. Because there were Official apps so far.

**This PR should be merged after https://github.com/YunoHost/yunohost-admin/pull/225**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ https://yunohost.org/apps
 
  - **official.json** contains the repository information of validated apps.
  - **community.json** contains all references to known YunoHost packages. If you want to add your app to the list, please [send a Pull Request](#contributing)
```
Comment (kay0u) [inline README.md]: After that, will community.json and official.json not be deleted?

Comment (maniackcrudelis) [reply README.md]: Not in a first time.
It was said during the meeting, but nothing decided, that we would wait at least the version 4 of YunoHost + 6 months.

Comment (kay0u) [reply README.md]: Ok

**Comment by maniackcrudelis** [comment]
Ready for reviews
@YunoHost/apps

**Comment by maniackcrudelis** [comment]
Replaced by https://github.com/YunoHost/apps/pull/727

**Pull request closed without merging**